src/components/Button/Button.scss

```diff
@@ -127,14 +127,10 @@ $partial-button-filled-pressed-box-shadow: inset 0 0 0 0 transparent,
 }
 
 .primary {
-  --p-button-color: var(--p-branded-action);
-  --p-button-color-hover: var(--p-branded-action-hovered);
-  --p-button-color-active: var(--p-branded-action-pressed);
```

Comment:
We should be setting `--p-button-text` here so that `critical` can set its own text color